### ISSUE TITLE
请问训练识别模型时没有报错直接终止是什么问题？

### ISSUE THREAD

**Issue by haohao2233** [PaddlePaddle/PaddleOCR#3473]
/opt/conda/envs/python35-paddle120-env/lib/python3.7/site-packages/paddle/fluid/layers/utils.py:26: DeprecationWarning: `np.int` is a deprecated alias for the builtin `int`. To silence this warning, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  def convert_to_list(value, n, name, dtype=np.int):
/opt/conda/envs/python35-paddle120-env/lib/python3.7/site-packages/skimage/morphology/_skeletonize.py:241: DeprecationWarning: `np.bool` is a deprecated alias for the builtin `bool`. To silence this warning, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  0, 1, 1, 0, 0, 1, 0, 0, 0], dtype=np.bool)
/opt/conda/envs/python35-paddle120-env/lib/python3.7/site-packages/skimage/morphology/_skeletonize.py:256: DeprecationWarning: `np.bool` is a deprecated alias for the builtin `bool`. To silence this warning, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  0, 0, 0, 0, 0, 0, 0, 0, 0], dtype=np.bool)
[2021/07/30 09:21:04] root INFO: Architecture : 
[2021/07/30 09:21:04] root INFO:     Backbone : 
[2021/07/30 09:21:04] root INFO:         model_name : large
[2021/07/30 09:21:04] root INFO:         name : MobileNetV3
[2021/07/30 09:21:04] root INFO:         scale : 0.5
[2021/07/30 09:21:04] root INFO:     Head : 
[2021/07/30 09:21:04] root INFO:         fc_decay : 0
[2021/07/30 09:21:04] root INFO:         name : CTCHead
[2021/07/30 09:21:04] root INFO:     Neck : 
[2021/07/30 09:21:04] root INFO:         encoder_type : rnn
[2021/07/30 09:21:04] root INFO:         hidden_size : 96
[2021/07/30 09:21:04] root INFO:         name : SequenceEncoder
[2021/07/30 09:21:04] root INFO:     Transform : None
[2021/07/30 09:21:04] root INFO:     algorithm : CRNN
[2021/07/30 09:21:04] root INFO:     model_type : rec
[2021/07/30 09:21:04] root INFO: Eval : 
[2021/07/30 09:21:04] root INFO:     dataset : 
[2021/07/30 09:21:04] root INFO:         data_dir : ./train_data/ic15_data
[2021/07/30 09:21:04] root INFO:         label_file_list : ['./train_data/ic15_data/rec_gt_test.txt']
[2021/07/30 09:21:04] root INFO:         name : SimpleDataSet
[2021/07/30 09:21:04] root INFO:         transforms : 
[2021/07/30 09:21:04] root INFO:             DecodeImage : 
[2021/07/30 09:21:04] root INFO:                 channel_first : False
[2021/07/30 09:21:04] root INFO:                 img_mode : BGR
[2021/07/30 09:21:04] root INFO:             CTCLabelEncode : None
[2021/07/30 09:21:04] root INFO:             RecResizeImg : 
[2021/07/30 09:21:04] root INFO:                 image_shape : [3, 32, 100]
[2021/07/30 09:21:04] root INFO:             KeepKeys : 
[2021/07/30 09:21:04] root INFO:                 keep_keys : ['image', 'label', 'length']
[2021/07/30 09:21:04] root INFO:     loader : 
[2021/07/30 09:21:04] root INFO:         batch_size_per_card : 2
[2021/07/30 09:21:04] root INFO:         drop_last : False
[2021/07/30 09:21:04] root INFO:         num_workers : 2
[2021/07/30 09:21:04] root INFO:         shuffle : False
[2021/07/30 09:21:04] root INFO:         use_shared_memory : False
[2021/07/30 09:21:04] root INFO: Global : 
[2021/07/30 09:21:04] root INFO:     cal_metric_during_train : True
[2021/07/30 09:21:04] root INFO:     character_dict_path : ppocr/utils/ic15_dict.txt
[2021/07/30 09:21:04] root INFO:     character_type : ch
[2021/07/30 09:21:04] root INFO:     checkpoints : None
[2021/07/30 09:21:04] root INFO:     debug : False
[2021/07/30 09:21:04] root INFO:     distributed : False
[2021/07/30 09:21:04] root INFO:     epoch_num : 40
[2021/07/30 09:21:04] root INFO:     eval_batch_step : [0, 10]
[2021/07/30 09:21:04] root INFO:     infer_img : doc/imgs_words_en/word_10.png
[2021/07/30 09:21:04] root INFO:     infer_mode : False
[2021/07/30 09:21:04] root INFO:     log_smooth_window : 20
[2021/07/30 09:21:04] root INFO:     max_text_length : 25
[2021/07/30 09:21:04] root INFO:     pretrained_model : ./pretrain_models/rec_mv3_none_bilstm_ctc_v2.0_train/best_accuracy
[2021/07/30 09:21:04] root INFO:     print_batch_step : 10
[2021/07/30 09:21:04] root INFO:     save_epoch_step : 5
[2021/07/30 09:21:04] root INFO:     save_inference_dir : None
[2021/07/30 09:21:04] root INFO:     save_model_dir : ./output/rec/ic15/
[2021/07/30 09:21:04] root INFO:     use_gpu : True
[2021/07/30 09:21:04] root INFO:     use_space_char : False
[2021/07/30 09:21:04] root INFO:     use_visualdl : False
[2021/07/30 09:21:04] root INFO: Loss : 
[2021/07/30 09:21:04] root INFO:     name : CTCLoss
[2021/07/30 09:21:04] root INFO: Metric : 
[2021/07/30 09:21:04] root INFO:     main_indicator : acc
[2021/07/30 09:21:04] root INFO:     name : RecMetric
[2021/07/30 09:21:04] root INFO: Optimizer : 
[2021/07/30 09:21:04] root INFO:     beta1 : 0.9
[2021/07/30 09:21:04] root INFO:     beta2 : 0.999
[2021/07/30 09:21:04] root INFO:     lr : 
[2021/07/30 09:21:04] root INFO:         learning_rate : 0.0001
[2021/07/30 09:21:04] root INFO:     name : Adam
[2021/07/30 09:21:04] root INFO:     regularizer : 
[2021/07/30 09:21:04] root INFO:         factor : 0
[2021/07/30 09:21:04] root INFO:         name : L2
[2021/07/30 09:21:04] root INFO: PostProcess : 
[2021/07/30 09:21:04] root INFO:     name : CTCLabelDecode
[2021/07/30 09:21:04] root INFO: Train : 
[2021/07/30 09:21:04] root INFO:     dataset : 
[2021/07/30 09:21:04] root INFO:         data_dir : ./train_data/ic15_data
[2021/07/30 09:21:04] root INFO:         label_file_list : ['./train_data/ic15_data/rec_gt_train.txt']
[2021/07/30 09:21:04] root INFO:         name : SimpleDataSet
[2021/07/30 09:21:04] root INFO:         transforms : 
[2021/07/30 09:21:04] root INFO:             DecodeImage : 
[2021/07/30 09:21:04] root INFO:                 channel_first : False
[2021/07/30 09:21:04] root INFO:                 img_mode : BGR
[2021/07/30 09:21:04] root INFO:             CTCLabelEncode : None
[2021/07/30 09:21:04] root INFO:             RecResizeImg : 
[2021/07/30 09:21:04] root INFO:                 image_shape : [3, 32, 100]
[2021/07/30 09:21:04] root INFO:             KeepKeys : 
[2021/07/30 09:21:04] root INFO:                 keep_keys : ['image', 'label', 'length']
[2021/07/30 09:21:04] root INFO:     loader : 
[2021/07/30 09:21:04] root INFO:         batch_size_per_card : 2
[2021/07/30 09:21:04] root INFO:         drop_last : True
[2021/07/30 09:21:04] root INFO:         num_workers : 2
[2021/07/30 09:21:04] root INFO:         shuffle : True
[2021/07/30 09:21:04] root INFO:         use_shared_memory : False
[2021/07/30 09:21:04] root INFO: train with paddle 2.0.2 and device CUDAPlace(0)
[2021/07/30 09:21:04] root INFO: Initialize indexs of datasets:['./train_data/ic15_data/rec_gt_test.txt']
W0730 09:21:04.186337  2350 device_context.cc:362] Please NOTE: device: 0, GPU Compute Capability: 7.0, Driver API Version: 10.1, Runtime API Version: 10.1
W0730 09:21:04.191743  2350 device_context.cc:372] device: 0, cuDNN Version: 7.6.
[2021/07/30 09:21:09] root INFO: load pretrained model from ['./pretrain_models/rec_mv3_none_bilstm_ctc_v2.0_train/best_accuracy']
eval model:: 100%|██████████████████████████| 1039/1039 [00:23<00:00, 43.99it/s]
[2021/07/30 09:21:32] root INFO: metric eval ***************
[2021/07/30 09:21:32] root INFO: acc:0.2532498796340876
[2021/07/30 09:21:32] root INFO: norm_edit_dis:0.7086016893146361
[2021/07/30 09:21:32] root INFO: fps:90.20009442693762

**Comment by LDOUBLEV** [comment]
你调用的是评估代码吧，上述信息里没有报错，是正常运行完退出了
你调用的命令是？
